src/resource-managers/expression-manager.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * Expression Manager module.
+ * @module resource-managers/expression-manager
+ * @see module:resource-managers/resource-manager
+ */
+'use strict';
+
+var ResourceManager = require('./resource-manager');
+
+/**
+ * Expression Manager Object
+ *
+ * Class representing an Expression manager.
+ * Expressions are formulas that execute on the SolveBio API. They return a
+ * single value or list of values with a specific data type.
+ *
+ * @constructor
+ * @augments ResourceManager
+ * @requires module:resource-managers/resource-manager
+ */
+
+var ExpressionManager = function(solveBio, expression, dataType, isList) {
```

Comment:
Actually, in a sense all the parameters are used in evaluation. Would it make more sense to just not use a resource for the base of this?